src/components/TvDetails/index.tsx

```diff
@@ -74,6 +75,8 @@ const messages = defineMessages({
   downloadstatus: 'Download Status',
   playonplex: 'Play on Plex',
   play4konplex: 'Play 4K on Plex',
+  playonjellyfin: 'Play on Jellyfin',
+  play4konjellyfin: 'Play 4K on Jellyfin',
```

Comment:
```suggestion
  play: 'Play on {mediaServer}',
  play4k: 'Play 4K on {mediaServer}',
```
We can just create a generic string that works for any type of media server here.

Then, you can do something like this:
```
intl.formatMessage(messages.play, { mediaServer: settings.currentSettings.mediaServerType === MediaServerType.PLEX ? 'Plex' : 'Jellyfin' })
```

Reply:
I like this! I went ahead and replaced the 'Jellyfin' with the actual name of the Jellyfin server.

Reply:
@acvigue I'm not sure if we should do that for now.  If we want to do that, we should also display the Plex server name.  I think we should consider that later.  Will need to get input from @sct.  I'm worried that longer server names might break the UI design on mobile, so we'll need to do more testing.

Reply:
Got it. Thanks for the input. @TheCatLady

Reply:
Reverted to original functionality ('plex' || 'jellyfin')